setup.py

```diff
@@ -28,19 +27,19 @@
     python_requires='>=3.5',
     install_requires=[
         'attr',
-        'beautifulsoup4>=4.6.3',
-        'clldutils',
-        'python-levenshtein',
-        'fuzzywuzzy',
+        'beautifulsoup4>=4.9.1',
+        'clldutils>=3.5.4',
+        'python-levenshtein>=0.12',
+        'fuzzywuzzy>=0.18',
     ],
     extras_require={
         'dev': ['flake8', 'wheel', 'twine'],
         'test': [
-            'mock',
-            'pytest>=5.4',
-            'pytest-mock',
-            'pytest-cov',
-            'coverage>=4.2',
+            'mock>=4.0.2',
```

Comment:
`mock` braucht man gar nicht mehr, wenn in der `setup.cfg` kein `mock_use_standalone_module` mehr vorkommt.

Reply:
but `pytest_clld/_app.py`'s line 5 imports mock thus it's needed - pytest-mock can be removed, I think